internal/controllers/router/tests/create-full/01-assert.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Router
+metadata:
+  name: create-full
+status:
+  resource: 
+    adminStateUp: true
+    name: create-full
+    status: ACTIVE
+    description: Router from "create full" test
+    tags:
+      - tag1
+      - tag2
+    adminStateUp: false
+  #  externalGateways:
```

Comment:
We should uncomment this and check that externalGateways is populated correctly.